Content.Shared/_RMC14/Pulling/RMCPullingSystem.cs

```diff
@@ -365,5 +380,27 @@ public override void Update(float frameTime)
 
             _pulling.TryStopPull(uid, pullable);
         }
+
+        var pullerQuery = EntityQueryEnumerator<PullerComponent, TransformComponent, CanFiremanCarryComponent>();
```

Comment:
This will iterate every entity able to pull even if they are not doing so currently, it would be better for performance to iterate something like <BeingPulledComponent, PullableComponent>, get what they are being pulled by, and run the code on that